browser/ui/views/tabs/brave_compound_tab_container.cc

```diff
@@ -342,7 +377,11 @@ BrowserRootView::DropTarget* BraveCompoundTabContainer::GetDropTarget(
     return nullptr;
   }
 
-  return GetTabContainerAt(loc_in_local_coords);
+  if (GetTabContainerAt(loc_in_local_coords)) {
+    return this;
+  }
+
+  return nullptr;
```

Comment:
Q. I'm wondering why this was changed.

Reply:
Aha, thanks for the explanation. That makes sense as layout routine are different between pinned and unpinned container and `TabContainerImpl` doesn't know about it.